crates/common/src/config/mod.rs

```diff
@@ -175,6 +181,9 @@ pub struct LightClientProverConfig {
     pub enable_recovery: bool,
     /// The starting DA block to sync from
     pub initial_da_height: u64,
+    /// Configuration for Risc0Host
+    #[serde(default)]
+    pub risc0_host_config: Risc0HostConfig,
```

Comment:
```suggestion
    pub risc0_host: Risc0HostConfig,
```

Parent struct already has the word config